administrator/language/en-GB/plg_quickicon_eos.ini

```
@@ -0,0 +1,21 @@
+; Joomla! Project
+; (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_QUICKICON_EOS="Quick Icon - Joomla End Of Support Notification"
```

```suggestion
PLG_QUICKICON_EOS="Quick Icon - Joomla! End Of Support Notification"
```
to be in sync with sys file